test/specs/wrapper/destroy.spec.js

```diff
@@ -61,4 +74,49 @@ describeWithShallowAndMount('destroy', mountingMethod => {
     const wrapper = mountingMethod(TestComponent)
     expect(() => wrapper.destroy()).toThrow()
   })
+
+  const StubComponent = { template: '<div><p></p></div>' }
+
+  ;[
```

Comment:
It would be great to use `jest.each` here, but that will fail in Karma :(

Reply:
A lot of cool jest things fail in karma... all good.